containers/verifier/Dockerfile.j2

```diff
@@ -7,10 +7,12 @@ COPY install-dependencies.sh                  /tmp/
 RUN  /tmp/install-dependencies.sh
 
 # Build virtualenv and concent apps
-COPY requirements.lock              /srv/http/concent_api/
-COPY middleman_protocol             /srv/http/middleman_protocol/
+COPY cacheable-requirements.lock    /srv/http/concent_api/
 COPY install-concent-api.sh         /tmp/
 RUN  /tmp/install-concent-api.sh
+COPY middleman_protocol                   /srv/http/middleman_protocol/
+COPY install-middleman-protocol.sh  /tmp/
+RUN  /tmp/install-middleman-protocol.sh
```

Comment:
`middleman_protocol` should be installed before Concent.

Reply:
This optimization separate `middleman_protocol` because it's always change. Why do you want to install it before Concent ? It's not possible(`depends on Concent requirements`) and cause that concent always will be installing.